src/newgrf_config.cpp

```diff
@@ -836,6 +836,10 @@ GRFConfig *GetGRFConfig(uint32 grfid, uint32 mask)
 	return nullptr;
 }
 
+const char* GetGRFName(const GRFFile* grffile)
```

Comment:
```suggestion
const char *GetGRFName(const GRFFile *grffile)
```
but,  I feel like this function could be removed entirely and its contents just rolled into the TrackDesc constructor

Reply:
Yep I agree! Updated.